src/cli/server/mod.rs

```diff
@@ -16,6 +16,7 @@ fn log_launch_and_open_browser(sc: &ServerConfig) {
     );
     if sc.graphiql() {
         let url = sc.graphiql_url();
+        let url = format!("https://tailcall.run/playground/?u={}/graphql", url);
```

Comment:
- [ ] Drop the `graphiql` setting.
- [ ] Always log url.
- [ ] Update documentation to enable cors.

Reply:
We can also rename the function to `log launch` because its not going to open the browser anymore.